hyppo/independence/hhg.py

```diff
@@ -42,8 +43,11 @@ class HHG(IndependenceTest):
         matrices. To call a custom function, either create the distance matrix
         before-hand or create a function of the form ``metric(x, **kwargs)``
         where ``x`` is the data matrix for which pairwise distances are
-        calculated and ``**kwargs`` are extra arguements to send to your custom
+        calculated and ``**kwargs`` are extra arguments to send to your custom
         function.
+    auto : boolean, default: False
```

Comment:
`auto` should be in the test function, like every other test in `hyppo`

Reply:
Changed.